scripts/split_eafs.py

```diff
@@ -126,13 +122,10 @@ def read_eaf(ie):
             }
             if 'PARTICIPANT' in params:
                 obj.speakerId = speaker_id
```

Comment:
This is actually an error, I'm guessing you've come from a JS background. Python objects aren't related to dictionaries in the way in which they are in JS. Specifically dictionary keys are not attributes. Dictionaries in python actually have no attributes for values so `obj.speakerId` will raise an AttributeError:
For example:
```
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> foo = {"a": 1, "b": 2}
>>> foo.a
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'dict' object has no attribute 'a'
```

Instead do
```python
obj["speakerId"] = speaker_id
```

Reply:
You got it - much time in JS and Angular land lately. Aiming to do some fundamental Python programming course to reset my basic assumptions.